test/Performance/setup_env_windows.ps1

```diff
@@ -1,46 +1,27 @@
 Param(
     [Parameter(Mandatory=$True,Position=1)]
-    [string]$PHP_VERSION,
-    [Parameter(Mandatory=$True,Position=2)]
-    [string]$PHP_THREAD,
+	[string]$PHP_ZIP,
+	[Parameter(Mandatory=$True,Position=2)]
+    [string]$SQLSRV_DRIVER,
     [Parameter(Mandatory=$True,Position=3)]
-    [string]$DRIVER_SOURCE_PATH,
-[Parameter(Mandatory=$True,Position=4)]
-    [string]$ARCH
+    [string]$PDO_DRIVER
     )
 
-IF($ARCH -ne "x64" -And $ARCH -ne "x86"){
-    Write-Host "ARCH must either x64 or x86"
-    Break
-}
-
-IF($PHP_THREAD -ne "nts" -And $PHP_THREAD -ne "ts"){
-    Write-Host "PHP_THREAD must either nts or ts"
-    Break
-}
-
-IF($PHP_VERSION -NotMatch "7.[0-1].[0-9]"){
-    Write-Host "PHP_VERSION must be in format of 7.x.x"
-    Break
-}
+$ErrorActionPreference = "Stop"
 
 $startingDir=$pwd.Path
 $tempFolder=Join-Path $startingDir "temp"
 
 Remove-Item temp -Recurse -Force -ErrorAction Ignore
 New-Item -ItemType directory -Path temp
 
-(New-Object System.Net.WebClient).DownloadFile("http://windows.php.net/downloads/releases/sha1sum.txt","$tempFolder\sha1sum.txt")
-$PHP70_LATEST_VERSION=type $tempFolder\sha1sum.txt | where { $_ -match "php-(7.0\.\d+)-src" } | foreach { $matches[1] }
-$PHP71_LATEST_VERSION=type $tempFolder\sha1sum.txt | where { $_ -match "php-(7.1\.\d+)-src" } | foreach { $matches[1] }
-
-$PHP_VERSION_MINOR=$PHP_VERSION.split(".")[1]
-
 Write-Host "Installing chocolatey..."
 iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
 Write-Host "Installing Git..."
 choco install -y git
-Set-Alias git 'C:\Program Files\Git\cmd\git.exe'
+
+$gitDir = 'C:\Program Files\Git\cmd\git.exe'
+#Set-Alias git 'C:\Program Files\Git\cmd\git.exe'

```

Comment:
I believe Set-Alias does update the path (not within this powershell session, but after you exit). But both ways work, I was just wondering.